### ISSUE TITLE
Can't copy chunk from conquest world to vanilla world

### ISSUE THREAD

**Issue by aaronegger** [Conquest-Reforged/ConquestReforged#266]
Hey guys,

I have created a world using the conquest generator (ConquestReforged-3.0.1-mc1.12.2).

Now i wanted to copy some chunks of the world to another world, that has a customized generator (not conquest) and now i get server crashes or if i try to copy only the Sections, Entities and TileEntities of a Chunk with the NBTExplorer, the whole junk gets destroyed.
Blocks are replaced with blocks from conquest and everything looks weird.

Normally copying these entries should never cause any issues, since there are only the ids, some attributes and positions of the blocks stored. At least for the TileEntities entry. 

If i deactivate the mod the server can start again. But i want to use the mod with the customized generated map and a backup of my house..

tl;dr
Can you help me transfering some chunks from a world generated with conquest to another world with not a conquest generator.

**Comment by HIPPOKLEIDES** [comment]
Why don't you make a schematic of what you want to bring into your new world?

**Comment by aaronegger** [comment]
I tried it (with mcedit2-win64-2.0.0-beta12).
Selected everything. Exported as schematic, imported it to the new map, set the position, confirmed and saved it.

Started the server. Server starts without any problems. I joined the game and travelled to the position of the imported area.
Then i get this: 
https://gist.github.com/aaronegger/f6ead4b3405c05f32e29bc577536e277

Some items were rendered before the server crashed an it looked as weird as copying the blocks.
Please guys, tell me what the conquest reforged world generator does that destorys all the chunks that are imported to a world that is generated with the original minecraft generator?

**Comment by HIPPOKLEIDES** [comment]
Mcedit 2 has been in beta for years and has always been unstable for me. I can't really recommend it. Helping with directly editing the data of world saves is a bit beyond the purpose of these tickets, so I'd still recommend just copying your build as a worldedit schematic and pasting it in with worldedit (mcedit, FAWE, and worldedit sometimes use different schematic formats).

**Comment by aaronegger** [comment]
Damn i am too stupid to read. You meant worldedit not mcedit.. I'll give that a try :+1: 
I didn't think about that worldedit supports exporting and importing stuff..

Sorry for the inconveniences. Thank you for your help :+1: 
Cheers :)
